src/App.js

```diff
@@ -32,13 +33,16 @@ function App() {
   useEffect(() => {
     const reactGa = gaInit(navigation);
     const mixpanel = mixpanelInit(navigation);
+    const security = new Security();
     navigation.subscribe(route => {
       if (route.type === 'ready') {
         console.debug(`[Mixpanel] Tracked: ${route.title}`);
         mixpanel.track('Pageview', { routeName: route.title });
 
         console.debug(`[GA] Tracked pageview: ${route.url.href}`);
         reactGa.pageview(route.url.href);
+
+        security.printConsoleInjectionWarning();
```

Comment:
It would be easy enough to wrap a production or mainnet check around this bit so that we don't have to deal with this warning in dev or testnet environments.  Let me know if that is desired.